site/book/01-getting-started/02-quickstart.md

```diff
@@ -86,14 +99,39 @@ You can preview the changes that will be made to the cluster:
 $ kpt live preview
 ```
 
-Finally, apply the resources to the cluster:
+Apply the resources to the cluster:
 
 ```shell
 $ kpt live apply --reconcile-timeout=15m
 ```
 
 This waits for the resources to be reconciled on the cluster by monitoring their status.
 
+## Fetch updates
+
+On Day n, there might be updates to the remote `nginx` package which you want to rebase your local package against.
+A typical `git rebase` might lead to merge conflicts in this scenario. `kpt pkg update` is semantic-aware
+and intelligently merges local changes with upstream updates.
+
+First, commit your local changes to `git` before update
+
+```shell
+git init; git add .; git commit -am "my-nginx"
```

Comment:
we should do an initial commit after the `get` command, so we can use 'git diff' to show changes during customization. So there's two commits:

Commit 1: Pristine fork
Commit 2: Cutomizatoins

and then later:

Commit 3: Updates

Reply:
Updated. Skipped Commit 3 though as it is not needed for the scope of this doc and to avoid too many `git` words in the doc.